spec/integration/hawkular_client_spec.rb

```diff
@@ -131,23 +125,24 @@ module Hawkular::Client::RSpec
         expect(resources1).to match_array(resources2)
       end
 
-      it 'Should both create and delete feed' do
-        feed_id1 = 'feed_1123sdn'
-        feed_id2 = 'feed_1124sdn'
-        @client.create_feed feed_id1
-        @hawkular_client.inventory_create_feed feed_id2
-
-        feed_list = @client.list_feeds
-        expect(feed_list).to include(feed_id1)
-        expect(feed_list).to include(feed_id2)
-
-        @client.delete_feed feed_id2
-        @hawkular_client.inventory.delete_feed feed_id1
-
-        feed_list = @hawkular_client.inventory_list_feeds
-        expect(feed_list).not_to include(feed_id1)
-        expect(feed_list).not_to include(feed_id2)
-      end
+      # FEED CREATION / DELETION CURRENTLY DISABLED
```

Comment:
Yep, this is related to @Jiri-Kremser comment above: what do we do for obsolete tests? If everybody here is fine about deleting them, I'll do it.